rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -445,6 +530,8 @@ private int retryGetEndStatus(URI endPath, boolean compensate) {
 
                 // 200 and 410 are the only valid response code for reporting the participant status
                 if (response.getStatus() == Response.Status.GONE.getStatusCode()) {
+                    // 412 implies that the participant did the correct thing and cleaned up so infer the status
```

Comment:
This comment seems unassertive to me. Does 412 "imply" or "require" that the participant did this thing. Also "infer" could imply an element of doubt, is it possible to express as "determine". Are we certain of the outcome here or are there cases where valid responses could be misinterpreted?

Reply:
I reworded the comment and referenced the specification